app/abilities/person_ability.rb

```diff
@@ -12,6 +12,8 @@ class PersonAbility < AbilityDsl::Base
   on(Person) do
     class_side(:index, :query).everybody
 
+    permission(:admin).may(:destroy)
```

Comment:
Da fehlt noch die Constraint Methode. Am besten gleich etwas in der Art von `everybody_except_herself` implementieren, dann braucht es den `not_self` Check im View Helper nicht mehr.